_papers/thesis-report/additional/extended-abstract.typ

```diff
@@ -100,11 +100,15 @@ The effectiveness of reading data in a decentralized environment has been increa
 In this work, we will similarly research how we can abstract data updates by using a query abstraction layer.
 The current (draft) Solid specification~@bib1:solid-spec describes each data store, or pod, as a document oriented interface where a user decides for each document who can access that document.
 Our goal is thus to create a query engine that effectively decides what document a resource should be stored in. Easily eliminating the access-path data dependency.
-We hypothesize that such a query engine has a 2x overhead in the number of HTTP requests and a 4x overhead in the execution time compared to a query engine that requires the user to configure the document explicitly. Such an overhead is acceptable since write speeds are, in contrast with read speeds, often not critical.  
+We hypothesize that such a query engine has a 2x overhead in the number of HTTP requests and a 4x overhead in the execution time compared to a query engine that requires the user to configure the document explicitly. Such an overhead is acceptable since write speeds are, in contrast with read speeds, often not critical.
+
+// TODO: Can you in a few words explain why this overhead is acceptable?"
 
 
 = Related Work
 
+// TODO: I'm missing related work around updates in distributed systems here, also CRDTs and so on. And I don't remember now, but do you mention these in the rest of your thesis?
```

Comment:
I don't talk about them since (as far as I know) they aren't that interesting. We have shards and replication, you mostly write to one copy and copies get synced in the background. I did not discuss this anywhere because those strategies are completely different from what Solid does. Do you think they are required? For the "book" I have my related work in "Semantic Web" and "Solid", I could add a part on Solid as how it differentiates itself from other Distributed DBMS?